packages/back-end/types/integrations/mssql.d.ts

```diff
@@ -0,0 +1,16 @@
+export interface MssqlConnectionParams {
+  user: string;
+  server: string;
+  database: string;
+  password: string;
+  port: number;
+  pool?: {
+    max?: number;
+    min?: number;
+    idleTimeoutMillis?: number;
```

Comment:
We don't want to use connection pools since GrowthBook Cloud is in a multi-tenant environment.  It's not safe to reuse connections across requests.